tests/Aspire.Hosting.Tests/Health/ResourceHealthCheckServiceTests.cs

```diff
@@ -129,12 +131,14 @@ public async Task HealthCheckIntervalDoesNotSlowBeforeSteadyHealthyState()
     {
         using var builder = TestDistributedApplicationBuilder.Create(testOutputHelper);
 
-        AutoResetEvent? are = null;
+        var channel = Channel.CreateBounded<DateTimeOffset>(3);
```

Comment:
IMO the channel could just be unbounded. I don't think providing a bounds is helpful to the test.